### PR TITLE
HM2

### PR DESCRIPTION
**Pull request by SergeyKravchuk**

### CODE REVIEW

File: README.md
Comment (pluhin) [inline README.md:1]: please revert this change, we need this file

**Review by pluhin — CHANGES_REQUESTED** [review]
Please squash commits, 37 commits is too much for one PR

**Review by SergeyKravchuk — COMMENTED** [review]
revert

**Comment by pluhin** [comment]
> > revert
> 
> Good, only squash is left

please make squash/fixup 
https://www.freecodecamp.org/news/git-squash-commits/

**Pull request closed without merging**